addons/account/views/audit_trail_report_views.xml

```diff
@@ -27,7 +27,21 @@
                 <field name="account_audit_log_move_id"/>
                 <field name="author_id"/>
                 <field name="date" string="Date"/>
-                <filter string="Update Only" name="update_only" domain="[('tracking_value_ids', '!=', False)]" groups="base.group_system"/>
+                <filter string="Updated Entries" name="audit_updated_entries" domain="[('tracking_value_ids', '!=', False)]" groups="base.group_system"/>
```

Comment:
That's a strange name but ok, maybe it makes sense from an audit point of view. "Contains update" would be more accurate but maybe less functionally sexy.?

Reply:
I don't have any strong opinion ;)
@tsb-odoo What do you think?